maps/mp/isnipe/extras/peoplealive.gsc

```diff
@@ -1,78 +1,72 @@
-/*Alive People counting script by iNuke for iSnipe 3*/
-
-#include common_scripts\utility;
-#include maps\mp\_utility;
 #include maps\mp\gametypes\_hud_util;
 
-CheckTeam()
-{
-
-if (level.alivecounterstarted != 1)
-{
-level.alivecounterstarted = 1;
-level.alliesalive = 0;
-level.axisalive = 0;
+/**
+ * Initialize the variables used by the peopleAlive module on a level.
+ */
+init() {
+    level.peopleAlive = [];
+    level.peopleAlive["allies"] = 0;
+    level.peopleAlive["axis"] = 0;
 }
 
-
-
-self.ateam = self.pers["team"];
-
-if (self.ateam == "allies")
-    {
-    level.alliesalive += 1;
-    } else if (self.ateam == "axis") {
-    level.axisalive += 1;
-    }
-
-self thread DoLabel();
-
-self waittill("death");
-
-if (self.ateam == "allies")
-    {
-    level.alliesalive -= 1;
-    } else if (self.ateam == "axis") {
-    level.axisalive -= 1;
-    }
-
+/**
+ * Initialize the variables used by the peopleAlive module on a player level.
+ */
+onPlayerConnected() {
```

Comment:
Method name `onPlayerConnected()`does not equal to method name called by moduleloader `onPlayerConnect()`

This throws:
```
******* script compile error *******
Error: unknown function onplayerconnect in maps/mp/isnipe/extras/peoplealive.gsc
************************************